### PR TITLE
0027 tink worker rearch

### PR DESCRIPTION
**Pull request by jacobweinstock**
## Description

rendered proposal: https://github.com/jacobweinstock/proposals/tree/0027-tink-worker-rearch/proposals/0027

## Why is this needed

Fixes: #

## Checklist:

I have:

- [ ] updated the documentation and/or roadmap (if required)
- [ ] added unit or e2e tests
- [ ] provided instructions on how to upgrade

-----
[View rendered proposals/0027/README.md](https://github.com/jacobweinstock/proposals/blob/0027-tink-worker-rearch/proposals/0027/README.md)

### CODE REVIEW

**Review by tstromberg — APPROVED** [review]

**Comment by mmlb** [comment]
Are we waiting on more discussion on this?

File: proposals/0027/README.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+- High unit test coverage
+- Documented design philosophy
+- Easy to understand and modify code
+- Build to be able to handle different container runtimes (containerd, kubernetes, etc)
+- Backward compatible CLI interaction
+- Support for public container registries
+- Support for a single container registry with authentication
+- Implement the global timeout (fixes [#198](https://github.com/tinkerbell/tink/issues/198))
+- Drop support for [emphemeral data](https://docs.tinkerbell.org/workflows/working-with-workflows/#ephemeral-data)
```
Comment (displague) [inline proposals/0027/README.md:36]:
```suggestion
- Drop support for [ephemeral data](https://docs.tinkerbell.org/workflows/working-with-workflows/#ephemeral-data)
```

**Comment by displague** [comment]
This seems well-intentioned. We've seen improvement on many of the points highlighted in this proposal with some points that we can still strive for.

@jacobweinstock Would you mind opening another PR to update this post-merge. We'll want to update the status and dates, but I think we can also show the progress that has been made by adding links to open issues reflecting the desired state of this proposal and also link to the closed issues/PRs.

**Comment by jacobweinstock** [comment]
This effort, unfortunately, has dropped off.

**Pull request closed without merging**